examples/Retrieval models (MovieLens 1M) - Youtube-DNN retrieval model example.ipynb

```diff
@@ -0,0 +1,437 @@
+{
```

Comment:
<p>I think that only train.head() makes the job, even for dask_cudf data frames</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>